src/ngx_http_lua_module.c

```diff
@@ -1037,8 +1037,8 @@ ngx_http_lua_create_loc_conf(ngx_conf_t *cf)
     /* set by ngx_pcalloc:
      *      conf->access_src  = {{ 0, NULL }, NULL, NULL, NULL};
      *      conf->access_src_key = NULL
-     *      conf->rewrite_src = {{ 0, NULL }, NULL, NULL, NULL};
-     *      conf->rewrite_src_key = NULL
+     *      conf->source = {{ 0, NULL }, NULL, NULL, NULL};
+     *      conf->source_key = NULL
      *      conf->rewrite_handler = NULL;
```

Comment:
I'm afraid these comments are wrong because `conf` no longer take the `source` and `source_key` fields directly (but via your newly defined `struct` as an intermediate field).